README.md

```diff
@@ -59,7 +59,7 @@ When a new version of Ghost comes out, you'll want to look over these [upgrade i
 
 You can talk to other Ghost users and developers in our [forum](https://forum.ghost.org/) (it's pretty awesome).
 
-New releases are announced on the [dev blog](http://dev.ghost.org/tag/releases/). You can subscribe by email or follow [@TryGhost_Dev](https://twitter.com/tryghost_dev) on Twitter, if you prefer your updates bite-sized and facetious. :saxophone::turtle:
+New releases are announced on the [dev blog](https://dev.ghost.org/tag/releases/). You can subscribe by email or follow [@TryGhost_Dev](https://twitter.com/tryghost_dev) on Twitter, if you prefer your updates bite-sized and facetious. :saxophone::turtle:
```

Comment:
https://dev.ghost.org/tag/releases/ can be replaced with https://blog.ghost.org/tag/releases/